### PR TITLE
Fixed Some typos.

### PR DESCRIPTION
**Pull request by NinjaCoderDev**
Fixed some typos in  readme.md and contributing.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ This project adheres to the Adobe [Code of Conduct](CODE_OF_CONDUCT.md). By part
 
 ## Ways of Contributing
 
-There are many ways of contributing, from testing and reporting an issue to suggesting and coding full components or features. Below is a summary of some of the best ways to get involved. 
+There are many ways of contributing, from testing and reporting an issue to suggest and coding full components or features. Below is a summary of some of the best ways to get involved.
```
Comment (richardhand) [inline CONTRIBUTING.md]: @NinjaCoderDev - this was correct before the change.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,11 +24,11 @@ This project adheres to the Adobe [Code of Conduct](CODE_OF_CONDUCT.md). By part
...
+There are many ways of contributing, from testing and reporting an issue to suggest and coding full components or features. Below is a summary of some of the best ways to get involved. 
 
 ### Joining Feature Discussions 💭
 
-You can add your voice to discussions around new and existing component features by commenting on an RTC. New components and features that openly invite public comment are marked by an [RTC](https://github.com/adobe/aem-core-wcm-components/labels/rtc) (Request to Comment) label. We strongly encourage users of the Core Components to bring their own project experience to these issues, as there may be alternative use-cases or requirements that haven't yet been considered.
+You can add your voice to discussions around new and existing component features by commenting on an RTC. New components and features that openly invite public comments are marked by an [RTC](https://github.com/adobe/aem-core-wcm-components/labels/rtc) (Request to Comment) label. We strongly encourage users of the Core Components to bring their own project experience to these issues, as there may be alternative use-cases or requirements that haven't yet been considered.
```
Comment (richardhand) [inline CONTRIBUTING.md]: plural isn't necessary here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -47,15 +47,15 @@ If you have thoughts on the code, the general architecture of the project, or wo
 1. Visit our [issue tracker on GitHub](https://github.com/adobe/aem-core-wcm-components/issues).
 1. File a `New Issue` as a `Bug Report`.
 1. Ensure your issue follows the [issue report guidelines](#issue-report-guidelines).
-1. Thanks for the report! The committers will get back to you in a timely manner, typically within one week.
+1. Thanks for the report! The committers will get back to you on time, typically within one week.
```
Comment (richardhand) [inline CONTRIBUTING.md]: the original fits better here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -47,15 +47,15 @@ If you have thoughts on the code, the general architecture of the project, or wo
...
 
 ### Requesting Features 🚀
 
 ##### Before Requesting a Feature
 
 * Have a quick search through the currently open [enhancement](https://github.com/adobe/aem-core-wcm-components/labels/enhancement) and [request to comment](https://github.com/adobe/aem-core-wcm-components/labels/rtc) issues to see if the idea has already been suggested. If it has, you may still have a slightly different requirement that isn't covered, in which case, feel free to comment on the open issue. 
 * Take a look at the [Core Components Roadmap](https://github.com/adobe/aem-core-wcm-components/wiki#roadmap) to see if your feature is already on the radar. If it is and doesn't have a public issue yet, feel free to create one, listing your own requirements.
-* Consider whether your requirement is generically useful rather than project-specific and would therefore benefit all users of the Core Components.
+* Consider whether your requirement is generically useful rather than project-specific and would, therefore, benefit all users of the Core Components.
```
Comment (richardhand) [inline CONTRIBUTING.md]: I don't think these commas are necessary here

File: content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md [hunk 5]
```diff
@@ -19,10 +19,10 @@ Extensible page component written in HTL.
 
 ## Features
 * Editable templates
-* Page title, subtitle, description and thumbnail
+* Page title, subtitle, description, and thumbnail
 * Navigation title, or hide from navigation
 * Vanity URL, page alias and redirection
-* Page tagging and define content language
+* Page tagging and define a content language
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md]: "Page tagging and content language definition"

File: content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md [hunk 6]
```diff
@@ -66,7 +66,7 @@ The following properties are written to JCR for this Page component and are expe
 7. `./subtitle` - defines a subtitle for this page
 8. `./jcr:description` - defines a description for this page
 9.  `./onTime` - defines when the page should become available
-10. `./offTime` - defines when the page should not be available any more
+10. `./offTime` - defines when the page should not be available anymore
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md]: "defines when the page should no longer be available"

File: content/src/content/jcr_root/apps/core/wcm/components/search/v1/search/README.md [hunk 7]
```diff
@@ -60,7 +60,7 @@ BLOCK cmp-search
 ```
 
 ## JavaScript Data Attribute Bindings
-Apply a `data-cmp-is="search"` attribute to the wrapper block to enable initialization of the JavaScript component.
+Apply a `data-cmp-is="search"` attribute to the wrapper block to enable the initialization of the JavaScript component.
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/search/v1/search/README.md]: prefer how it reads here without this change :)

File: content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md [hunk 8]
```diff
@@ -19,10 +19,10 @@ Extensible page component written in HTL.
 
 ## Features
 * Editable templates
-* Page title, subtitle, description and thumbnail
+* Page title, subtitle, description, and thumbnail
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md]: this new style, if introduced, should be applied to all lists. As it works fine without, I would prefer the original style of no serial comma.

File: content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md [hunk 9]
```diff
@@ -49,10 +49,10 @@ The Page component uses the following use objects:
 ## Component Policy Configuration Properties
 The following configuration properties are used:
 
-1. `./clientlibs` - allows definition of a list of client libraries to be loaded by the pages associated with this policy configuration
-2. `./clientlibsJsHead` - allows definition of client libraries for which JavaScript is specifically intended to be loaded
+1. `./clientlibs` - allows the definition of a list of client libraries to be loaded by the pages associated with this policy configuration
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/README.md]: Prefer how these read without the introduced "the"

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -123,7 +123,7 @@ Once an issue is reported, the project committers will assign it a relevant labe
...
+By contributing your code to the Adobe Marketing Cloud Github Organisation you grant Adobe a non-exclusive, irrevocable, worldwide, royalty-free, sublicensable, transferable license under all of Your relevant intellectual property rights (including copyright, patent, and any other rights), to use, copy, prepare derivative works of, distribute and publicly perform and display the Contributions on any licensing terms, including without limitation: (a) open source licenses like the Apache License, Version 2.0; and (b) binary, proprietary, or commercial licenses. Except for the licenses granted herein, You reserve all right, title, and interest in and to the Contribution.
```
Comment (richardhand) [inline CONTRIBUTING.md]: Seems to be a diff here, not sure what changed?

Comment (NinjaCoderDev) [reply CONTRIBUTING.md]: Spelling of 'sublicenseable' was wrong, so changed it to 'sublicensable'.

**Comment by richardhand** [comment]
Thanks @NinjaCoderDev for your contribution.

**Pull request merged**